### ISSUE TITLE
AVT3: All the graphics are not read out by JAWS

### ISSUE THREAD

**Issue by xiepingp** [ibm-cloud-docs/solution-tutorials#66]
=====ENVIRONMENT=====
IBM CI 162 Checkpoint Violation: IBM 1.3.1 Info and Relationships
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html
Test Tool Type: JAWS Screen Reader 2019

=====DEFECT DESCRIPTION=====
All the graphics are not read by JAWS in Continuous Deployment to Kubernetes page

=====STEPS TO REPRODUCE=====
Page location: https://cloud.ibm.com/docs/Registry?topic=solution-tutorials-continuous-deployment-to-kubernetes
Keep JAWS running and reading all the contents 
-> 9 graphics in this page are not read by JAWS
![Screen Shot 2020-08-25 at 12 30 42 PM](https://user-images.githubusercontent.com/63960906/91123368-447ec980-e6cf-11ea-981d-ad1320a8a81f.png)
![Screen Shot 2020-08-25 at 12 31 14 PM](https://user-images.githubusercontent.com/63960906/91123380-49dc1400-e6cf-11ea-9c8f-8db2d2fae882.png)

===== EXPECTED RESULT =====
The intent of this Success Criterion is to ensure that information and relationships that are implied by visual or auditory formatting are preserved when the presentation format changes.
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Comment by data-henrik** [comment]
This has been addressed and will be available with the next doc updates

**Comment by xiepingp** [comment]
Retest the issue and now JAWS reads all the graphics as expected.
